src/LayerPlan.h

```diff
@@ -477,7 +477,7 @@ class LayerPlan : public NoCopy
      * \param p The point to travel to.
      * \param force_comb_retract Whether to force a retraction to occur.
      */
-    GCodePath& addTravel(const Point p, const bool force_retract = false);
+    GCodePath& addTravel(const Point p, const bool force_retract = false, const bool unretract_before_last_travel_move = false);
```

Comment:
Documentation should also be updated (it's already outdated, since it mentiones force_comb_retract)